server/apps/main/templates/main/help.html

```diff
@@ -14,31 +14,131 @@
         <div class="list-group col-lg-3 cotent-list" id="content-bar">
           <h5 class="list-group-item content-title">Contents</h5>
           <a class="list-group-item list-group-item-action  d-flex justify-content-between align-items-center"
-              href="#placeholder_1">Placeholder 1<span class="badge badge-pill"></span></a>
+            href="#what-stories">What stories should I share on AutSPACEs?<span class="badge badge-pill"></span></a>
           <a class="list-group-item list-group-item-action  d-flex justify-content-between align-items-center"
-              href="#placeholder_2">Placeholder 2<span class="badge badge-pill"></span></a>
+            href="#who-can-share">Who can share stories on AutSPACEs?<span class="badge badge-pill"></span></a>
+          <a class="list-group-item list-group-item-action  d-flex justify-content-between align-items-center"
+            href="#what-happens-to-stories">What happens to the stories I share on AutSPACEs?<span
+              class="badge badge-pill"></span></a>
+          <a class="list-group-item list-group-item-action  d-flex justify-content-between align-items-center"
+            href="#how_to_decide">How do you decide which stories to publish on the AutSPACEs website?<span
```

Comment:
```suggestion
            href="#how-to-decide">How do you decide which stories to publish on the AutSPACEs website?<span
```
Sorry to be such a pedant, but one last case.

Reply:
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO